Language review of beginners.md (#2434)
travis-ci/docs-travis-ci-com: user/for-beginners.md

Our [Common Builds Problems](/user/common-build-problems/) page is a good place
to start troubleshooting why your build is broken.

## Infrastructure and Environment Notes

Travis CI offers a few different infrastructure environments, so you can select
the setup that suits your project best:

> * *Ubuntu Linux* - these Linux Ubuntu environments run inside full virtual machines, provide plenty of computational resources, and support the use of `sudo`, `setuid`, and `setgid`.

Reviewer: Shall we add a link to

> * *macOS* - uses one of several versions of the macOS operating system. This environment is useful for building projects that require the macOS software, such as projects written in Swift. It is not a requirement to use the macOS environment, if you develop on a macOS machine.

Reviewer: I think it'd be helpful for us to link to the macOS reference page here, https://docs.travis-ci.com/user/reference/osx/, wdyt?

Our [Common Builds Problems](/user/common-build-problems/) page is a good place
to start troubleshooting why your build is broken.

## Infrastructure and Environment Notes

Travis CI offers a few different infrastructure environments, so you can select
the setup that suits your project best:

> * *Ubuntu Linux* - these Linux Ubuntu environments run inside full virtual machines, provide plenty of computational resources, and support the use of `sudo`, `setuid`, and `setgid`.

Reviewer: now that we have the new Linux page let's add it here as a link too :)

> * *Ubuntu Linux* - these Linux Ubuntu environments run inside full virtual machines, provide plenty of computational resources, and support the use of `sudo`, `setuid`, and `setgid`. For more information on the available Ubuntu Linux Build Environments, see [here](/user/reference/linux/).

Reviewer: ```suggestion
* *Ubuntu Linux* - these Linux Ubuntu environments run inside full virtual machines, provide plenty of computational resources, and support the use of `sudo`, `setuid`, and `setgid`. Check out more information on the [Ubuntu Linux Build Environment](/user/reference/linux/).
```

* *macOS* - uses one of several versions of the macOS operating system. This environment is useful for building projects that require the macOS software, such as projects written in Swift. It is not a requirement to use the macOS environment, if you develop on a macOS machine. Here you can find more details on the [macOS Build Environment](/user/reference/osx/).

> * *Windows* - currently only Windows Server version 1803 is supported. If you want to know more about it, see the [Windows Build Environment](/user/reference/windows/).

Reviewer: ```suggestion
* *Windows* - currently Windows Server version 1803 is supported. If you want to know more about it, see the [Windows Build Environment](/user/reference/windows/).
```

I'd remove the only since I believe it's diminishing the phrase, wdyt?

> More details on our environments are available in our [CI Environment](/user/ci-environment/) documentation.

Reviewer: ```suggestion
More details on our build environments are available in our [CI Environment](/user/ci-environment/) documentation.
```
